pkg/csv/reader.go

```diff
@@ -14,10 +14,12 @@ type CSVReader struct {
 	CSVConfig *config.CSVConfig
 	reader    *csv.Reader
 	lineNum   uint64
+	Comma     rune
```

Comment:
If CSVConfig stored delimiter, this field would be needless.

Reply:
you need think all csv file have one common format

Reply:
> you need think all csv file have one common format

Each file will have their owned csv reader, and the delimiter option has been configured in `CSVConfig`, why not to use it directly?